README.md

```diff
@@ -45,6 +45,7 @@ _The `GITHUB_TOKEN` secret is [automatically generated by GitHub](https://help.g
   with:
     githubToken: ${{ secrets.GITHUB_TOKEN }}
     jpegQuality: "80"
+    jpegProgressive: false

```

Comment:
@benschwarz I have this default to false, let me know if you want this to default to true.

Reply:
Default being false is good. 👍